app/Http/Controllers/ProjectCommentController.php

```diff
@@ -50,7 +50,13 @@ public function edit(ProjectComment $comment)
             return redirect('/')->withErrors('You are not authorized to do this action!');
         }
 
-        return view('project_comments.edit', compact('comment'));
+        $route = 'comments';
+        $object = 'message';
+        $routeToDelete = '/comments/'.$comment->id;
```

Comment:
it's because the edit view is used by chat messages edition and project comments edition. If you prefere I can use a helper view ?